_layouts/email.html

```diff
@@ -42,13 +46,20 @@ <h2>{{ page.name }}</h2>
 {% include list.html %}
 
 <script type="text/javascript">
-	function openEmail () {
+	function openEmail(client) {
 		const subject = encodeURIComponent(`{{ page.subject }}`.trim());
-        const body = encodeURIComponent(`{{ page.body }}`.trim());
-        const recipients = `{{ page.recipients | join: ',' }}`;
-        const cc = `{{ page.cc | join: ',' }}`;
-        const ccText = cc !== null && cc.length > 0 ? `cc=${cc}&` : ``;
-        location.href = `mailto:${recipients}?${ccText}subject=${subject}&body=${body}`;
+    const body = encodeURIComponent(`{{ page.body }}`.trim());
+    const recipients = `{{ page.recipients | join: ',' }}`;
+    const cc = `{{ page.cc | join: ',' }}`;
+    const ccText = cc !== null && cc.length > 0 ? `cc=${cc}&` : ``;
+
+    if (client === 'mailto') {
+      location.href = `mailto:${recipients}?${ccText}subject=${subject}&body=${body}`;
+    } else if (client === 'gmail') {
+      location.href = `https://mail.google.com/mail/?view=cm&fs=1&to=${recipients}&${ccText}su=${subject}&body=${body}`;
+    } else if (client === 'yahoo') {
+      location.href = `https://compose.mail.yahoo.com/?to=${recipients}&${ccText}subj=${subject}&body=${body}`;
+    }
```

Comment:
these clients should be enumerated somewhere so they can be referenced from the function call as well. declaring as literals is dangerous.